src/Template/Element/Form/relations.twig

```diff
@@ -54,7 +61,24 @@
                         </div>
                     </div>
 
-                    <div v-if="!loading && objects.length === 0 && addedRelations.length === 0"  class="emptyResults">
+                    <div class="list-actions">
+                        {# <button class="show-more" @click.prevent="showMoreRelated()">Show More</button> #}
+
+                        <select class="page-size-selector" v-model="step">
+                            <option v-for="size in pageSizeOptions"><% size %></option>
+                        </select>
+                        <button @click.prevent="firstPage()">first</button>
+                        <button @click.prevent="prevPage()">prev</button>
+                        <button @click.prevent="nextPage()">next</button>
+                        <button @click.prevent="lastPage()">last</button>
+                    </div>
```

Comment:
Use translator function for pure text, i.e.:
```
<button @click.prevent="firstPage()">{{ __('first') }}</button>
<button @click.prevent="prevPage()">__('prev')</button>
<button @click.prevent="nextPage()">__('next')</button>
<button @click.prevent="lastPage()">__('last')</button>
```

Reply:
thanks, i wasn't aware of that